controllers/update_config_map.go

```diff
@@ -48,7 +48,7 @@ func (u updateConfigMap) reconcile(ctx context.Context, r *FoundationDBClusterRe
 	existing := &corev1.ConfigMap{}
 	err = r.Get(ctx, types.NamespacedName{Namespace: configMap.Namespace, Name: configMap.Name}, existing)
 	if err != nil && k8serrors.IsNotFound(err) {
-		logger.Info("Creating config map")
+		logger.Info("Creating config map", "name", configMap.Name)
```

Comment:
That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.